ftw/subsite/viewlets/subsitelogoviewlet.py

```diff
@@ -18,13 +19,15 @@ def __init__(self, context, request, view, manager=None):
     def update(self):
         super(SubsiteLogoViewlet, self).update()
 
-        portal = self.portal_state.portal()
+        portal = api.portal.get_navigation_root(self.context)
```

Comment:
I propose to rename `portal` to `navigationroot` or short `navroot` because a subsite is actually not the portal.